### PR TITLE
EWS: Added email reports balancing script

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo

### CODE REVIEW

**Comment by gcapalbo** [comment]
thanks Kamil, looks good.  if possible could we add two more checks:

- could we check the number of reports in the source vs. the number of reports in the target. i realize some time has passed so these might not match up exactly, but if they're off by a lot it should alert us to an error
- could we also check to make sure that each report either has the send_to_owner set to True or recipient_emails that is not empty, just to make sure that the fix for send_to_owner was applied

**Comment by kkrampa** [comment]
@gcapalbo Updated. Additionally I removed unused views from ews config page to make sure that other buttons won't be clicked by accident.

**Comment by gcapalbo** [comment]
looks good, thanks

File: custom/ewsghana/balance.py [hunk 1]
```diff
@@ -243,3 +246,94 @@ def balance_migration(self, date=None):
...
+                if sql_location.is_archived:
+                    desc += 'Location {} is not active; '.format(location_code)
+            except SQLLocation.DoesNotExist:
+                desc += 'Location {} does not exist; '.format(location_code)
+
+            migration_problem, _ = EWSMigrationProblem.objects.get_or_create(
+                domain=self.domain,
+                object_id=username,
+                object_type='email_report',
+                external_id=external_id
```
Comment (gcapalbo) [inline custom/ewsghana/balance.py]: I just realized, external_id is defined as varchar(128) - will the external_id ever exceed 128 characters or should it always fit?

Comment (kkrampa) [inline custom/ewsghana/balance.py]: I was testing this using data from ews v1 and it always fit.

**Pull request merged**